extensions/landing-pages.rb

```diff
@@ -21,22 +22,24 @@ def after_configuration
 
       # Generate all Landing Pages
       def proxy_landing_pages(locale_obj)
-        process_collection(locale_obj[:lang]).each do |model|
+        process_collection(locale_obj).each do |model|
           app.proxy(
             path_for_proxy(model.url_slug, locale_obj[:id]),
-            "/landing-pages/template.html",
+            '/landing-pages/template.html',
             locals: model.merge({ locale_obj: locale_obj }).with_indifferent_access,
             layout: model.layout || 'layout',
             ignore: true
           )
         end
       end
 
-      def process_collection(lang)
-        collection = app.data['landing-pages'].pages
-        collection = collection.map{ |tuple| tuple[1] } # contentful passes ["id", { ... }]
-        collection = collection.map{ |model| localize_entry(model, lang, default_locale_obj[:lang]) }
-        return collection
+      def process_collection(locale_obj)
```

Comment:
Only real question is could we use this `MiddlemanLandingPagesHelpers::landing_pages_collection` (given we use that in another place).